docs/tutorials/posts/llama-nemotron-super-v1.5-evals.md

```diff
@@ -164,7 +178,26 @@ ns eval \
                     --enable-auto-tool-choice"
 ```
 
+#### Command for RULER Eval (Reasoning on)
 
+For RULER we need to use the same `data_dir` in the evaluation command as we used in the data preparation. We also
+need to use the data preparation `setup` as part of the benchmark name. Finally it's important not to specify
+`++inference.tokens_to_generate` as RULER has a fixed value of this parameter for each task.
+
+```bash
+ns eval \
+    --cluster=local \
+    --model=/workspace/Llama-3_3-Nemotron-Super-49B-v1_5 \
+    --server_type=vllm \
+    --output_dir=/workspace/llama_nemotron_49b_1_5_ruler/ \
+    --benchmarks=ruler.nemotron_super_128k \
+    --data_dir=/workspace/ns-data \
+    --server_gpus=2 \
+    ++inference.tokens_to_generate=65536 \
```

Comment:
Doesn't this contradict the above statement that `inference.tokens_to_generate` should not be supplied?